server/sonar-server/src/main/java/org/sonar/server/app/ServerProcessLogging.java

```diff
@@ -88,10 +88,9 @@ public LoggerContext configure(Props props) {
     LoggerContext ctx = helper.getRootContext();
     ctx.reset();
 
-    helper.enableJulChangePropagation(ctx);
-
     configureRootLogger(props);
     helper.apply(logLevelConfig, props);
+    helper.enableJulChangePropagation(ctx);
```

Comment:
I think this should not have any impact, however since we are enabling all levels for JUL (via Propagator), I think it's better to configure the logger and the correct filter before removing all JUL filters.